community/code-of-conduct.md

```diff
@@ -0,0 +1,166 @@
++++
+
+toc=true
+
++++
+
+# [Name Option 1] Workplace Harassment - Code of Conduct 
+# [Name Option 2] An informal guide to avoid and deal with events of harassment and bullying
+# [Name Option 3] Workplace Harassment - An informal guide
+
+## Some basic fundamentals
+
+1. Humans are flawed and make mistakes on a regular basis while discovering new ways to hurt new people.
+
+2. We don't live in an ideal world, so we are not dealing with ideal scenarios.
+
+3. But the above statement doesn't provide wind to the practicality argument. Practicality argument is an exceptionally dangerous proposition in itself. So any time any particular statement is backed up by "being practical", it should take care of few things :-
+   1. Being practical means what can be done or is possible in some finite duration. So it should be aggressively progressive in nature and it shouldn't settle on "n" but "n + ∞". We can start from "n" but the vision should be clear to reach "n + ∞" as fast as possible.
+   2. It should be “enabler” in nature.
+
+4. Any democratic argument in case of harassment should be thrown out of the window immediately. For example, "9 out of 10 people are okay with this".
+
+5. Humans are not rational, so don't expect them to behave rationally. We are totally full of feelings.
+
+6. We don't accept snowflake argument. Some people have thick skin and are borderline stoic, and they think that "I am responsible for my feelings" and on the opposite end of emotional spectrum, are people who think that "You don't get to decide if you hurt someone". We will try to have an inclusive space for both of these people.
+
+7. Humans hate to forgive people if forgiving is tough.
+
+8. Convincing a human that they are wrong or have done wrong is exceptionally tough.
+
+9. Humans have double standards. Deal with it.
+
+## Basic Protocol in case you are harassed/bullied/found something offensive/weird/unacceptable
+
+Subject -> Who has been harassed/bullied
+
+Type 1 :- Constitutionally Legal Offense
+
+1. In this case the team is requested not to micromanage. let it go through official channels of college and judiciary and let them deal with it.
+
+Type 2 :- Everything Else
+
+1. If it has happened for the first time, dm the person. And let {pronoun} know that this behaviour is offensive to the subject. If not comfortable confronting then **????????????? @team comments**
```

Comment:
Just use "they" instead of {pronoun}. "They" includes everyone.

Reply:
I think we should add a `0` step - get to know the incident from both the guilty and the victim. They'll tell their own versions though it'll be helpful in avoiding false positives or at least give a fair chance to multiple perspectives of the same incident to come forth.